.travis/build.sh

```diff
@@ -2,4 +2,4 @@
 
 chmod u+x gradlew
 
-./gradlew clean build minJosmVersionClasses
+./gradlew clean build
```

Comment:
You can add task `compileJava_minJosm`, which is the new name for what was `minJosmVersionClasses` in previous versions of the gradle-josm-plugin.
It checks if the plugin can be compiled against the minimum JOSM version that is given in the manifest.